description: use system zlib library instead of embedded lzma in courgette
author: Michael Gilbert <mgilbert@debian.org>

--- a/courgette/BUILD.gn
+++ b/courgette/BUILD.gn
@@ -60,7 +60,6 @@ static_library("courgette_lib") {
 
   deps = [
     "//base",
-    "//third_party/lzma_sdk",
   ]
 
   public_deps = [
@@ -81,7 +80,6 @@ source_set("courgette_common") {
   ]
   deps = [
     "//base",
-    "//third_party/lzma_sdk",
   ]
 }
 
--- a/courgette/crc.cc
+++ b/courgette/crc.cc
@@ -7,6 +7,8 @@
 #include <stdint.h>
 #include <stddef.h>
 
+#define COURGETTE_USE_CRC_LIB
+
 #ifdef COURGETTE_USE_CRC_LIB
 #  include "zlib.h"
 #else
--- a/third_party/perfetto/gn/BUILD.gn
+++ b/third_party/perfetto/gn/BUILD.gn
@@ -323,7 +323,6 @@ if (enable_perfetto_trace_processor || e
         "//buildtools:zlib",
       ]
     } else {
-      public_configs = [ "//third_party/zlib:zlib_config" ]
       public_deps = [
         "//third_party/zlib",
       ]
